codespell_lib/data/dictionary.txt

```diff
@@ -16733,6 +16739,7 @@ sould->could, should, sold,
 souldn't->shouldn't
 soundard->soundcard
 sountrack->soundtrack
+sourbraten->sauerbraten
```

Comment:
German? I don't know what the consensus on this is, @larsoner ?

Reply:
removed

Reply:
Seems to exist in English language as a loanword, also
https://en.wikipedia.org/wiki/Sauerbraten

... and there are lots of them.
https://en.wikipedia.org/wiki/Lists_of_English_words_by_country_or_language_of_origin

I guess it is legitimate to include them in the dictionary.